HeterogeneousCore/AlpakaInterface/interface/binarySearch.h

```diff
@@ -0,0 +1,29 @@
+#ifndef HeterogeneousCore_AlpakaInterface_interface_binarySearch_h
```

Comment:
was there a comment clearly asking to put math functions in HeterogeneousCore/AlpakaInterface?

Reply:
Yes: https://github.com/cms-sw/cmssw/pull/45117#discussion_r1674782508